submissions/BogdanGapon/HTML & CSS popup/index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <nav class="header-nav">
+          <ul class="header-nav__list">
+            <li class="header-nav__item">
+              <a href="" class="header-nav__link">Mail</a>
+            </li>
+            <li class="header-nav__item">
+              <a href="" class="header-nav__link" target="_blank">Images</a>
+            </li>
+            <li class="header-nav__item popup-item">
+              <label class="custom-checkbox" for="check"></label>
+              <input
+                class="checkbox visually-hidden"
+                type="checkbox"
+                id="check"
+              />
+              <span class="span"></span>
+              <div class="popup-wrapper">
+                <div class="popup-flex-wrapper">
```

Comment:
Please, make all icon list semantic - use `ul>li` tags

Reply:
ahah it`s a joke?? previous mentor say to delete all ul>>li

Reply:
Please, delete all alt attrtibutes **values** in popUP and nav icons (img tags). If image is used for decorations and don't have any sanse or information for user, alt attribute should be empty, without any symbols or spacing.

Reply:
Values of alt attributes. <img src="./image.png" alt=""/>